### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by vaedprasad** [meredithdobrzynski/google_play_project#14]
The project “Predicting Google Play Store App Ratings” aims to identify which characteristics of an application, such as category, memory size, or price, yield the highest ratings. This group decided to focus on ratings rather than downloads in order to isolate their performance metric for user satisfaction rather than mere popularity or marketing budget. In order to select the feature space, the group leverages a correlation plot to find the most highly correlated features.

One aspect of the final report that I like is the decision to explore different loss functions when fitting the model, including L1 Loss, Quadratic Loss, and Huber Loss. I also agree with the decision to use one-hot encodings for all categorical features, which clearly improved the model’s accuracy by reducing training error by 3.4%. I also like the visualizations the group included in each model to easily understand how each model was predicting relative to the true values.

One potential area for improvement is that when cleaning the data, the group dropped two rows for missing values. I would have been curious if it would have been possible to impute the missing data here. In addition, in the Weapons of Math Destruction section the group claims that an application’s rating is harmless; however, this might be rather naïve. A model that can predict which characteristics of an application yield the highest ratings may result in developers trying to optimize for these metrics rather than customer performance, which may hurt user sentiment for applications. In addition, it may allow larger companies that can develop these models to continue to optimize and out-perform smaller companies. Finally, I would have liked to have seen the group experiment with non-linear models such as neural networks and K-nearest neighbors, as it is presumptuous to claim that the ideal model for such a complicated feature space is a linear one.
